item/hafdon; zorq.json

```diff
@@ -0,0 +1,82 @@
+				"The blanket may only be used in this way for 10 minutes before it loses its efficacy. The blanket regains its efficacy at the next dawn."
+			]
+			
+			
+		},
+		{
+			"name": "Figurine of Wondrous Power, Onyx Wolf",
+			"wondrous": true,
+			"tier": "Major",
+			"rarity": "Rare",
+			"source": "zorq",
+			"page": null,
+			"entries": [
+				"A figurine of wondrous power is a statuette of a beast small enough to fit in a pocket. If you use an action to speak the command word and throw the figurine to a point on the ground within 60 feet of you, the figurine becomes a living creature. If the space where the creature would appear is occupied by other creatures or objects, or if there isn't enough space for the creature, the figurine doesn't become a creature.",
+				"The creature is friendly to you and your companions. It understands your languages and obeys your spoken commands. If you issue no commands, the creature defends itself but takes no other actions.",
+				"The creature exists for a duration specific to each figurine. At the end of the duration, the creature reverts to its figurine form. It reverts to a figurine early if it drops to 0 hit points or if you use an action to speak the command word again while touching it. When the creature becomes a figurine again, its property can't be used again until a certain amount of time has passed, as specified in the figurine's description.",
+				"{@creature wolf||Onyx Wolf}.",
+				"This onyx statuette of a dog can become a {@creature wolf} for up to 6 hours. The {@creature wolf} has an Intelligence of 8 and can speak Common. It also has dark vision out to a range of 60 feet and can see {@condition invisible} creatures and objects within that range. Once it has been used, it can't be used again until 7 days have passed."
+			],
+			"srd": true
+		},
+		{
+			"name": "Three School Mask",
+			"wondrous": true,
+			"weight": 1,
+			"rarity": "Rare",
+			"source": "zorq",
+			"page": null,
+			"entries": [
+				"text",
+				{
+					"name": "Donning the mask",
+					"entries": [
+						"The mask is nigh indestructible. When the creature creature wearing the mask is destroyed, the mask usually survives and breaks free. On its own, the object detects as magical with mixed enchantment and necromantic auras. It is a tempting souvenir, but anyone foolish enough to don the mask is immediately wracked by pain and takes 7 ({@dice 2d6}) necrotic damage. The character must make a successful DC 15 Wisdom saving throw or become dominated by the mask. The domination arrives slowly. The character acts normally for a day or two, but then the character notices periods of time that cannot be accounted for. During these times, the character [...]. This process takes a week, after which the character is freed from domination as the undead creature is reborn."
+					]
+				}
+			],
+			"charges": 3,
+			"attachedSpells": [
+				"animal friendship"
+			],
+			"recharge": "dawn",
+			"_isEnhanced": true,
```

Comment:
Properties beginning with `_` are (generally) temporary and shouldn't be included in the data

Reply:
Gotcha, thanks.